Clients.md

```diff
@@ -18,7 +17,7 @@ Authored by: ShowierData9978 <contact@showierdata.xyz> on 4/18/24 (CT).
 
 - Cache chat and post data
 - Use Cloudlink to get updates to cached information
-- You must use V1 of the WebSocket Server
+- You must use the latest version of the API or Websocket.
```

Comment:
Remove "must", shouldn't be a requirement. There is also many APIs, and technically the WebSocket could be seen as an API, so I'd just make it more generic to cover every API.

Reply:
aaaand github did not include the commit suggestion thing

Reply:
oh wait
that was for the specific commit nvn